scripts/utilities/update_revision.sh

```diff
@@ -150,7 +160,13 @@ get_latest_revision ()
 		if [[ $(expr index "${branch}" "HEAD") -ne 0 ]]; then
 			branch="HEAD"
 		else
+		    # Since GCC moved to git (January 2020), the name of the branch
+		    # has looked like: ibm/<branch>
+		    if [[ $package == "gcc" ]] && [[ $configset != "next" ]]; then
```

Comment:
Why is it necessary to exclude AT-next here?

Reply:
AT next is feeded with the master branch, not an IBM branch.

Reply:
Ok, thanks for the clarification =)